scripts/commands/checkOrphanPages.ts

```diff
@@ -121,6 +123,7 @@ async function determineTocFiles(args: Arguments): Promise<string[]> {
     globs.push(
       "docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/[0-9]*/_toc.json",
     );
+    ``;
```

Comment:
Tip before opening up PRs to review your PR proactively to check for issues like this

```suggestion
```

Reply:
Duly noted, I appreciate your patience in reviewing these :)